gradle.properties

```diff
@@ -8,3 +8,5 @@ org.gradle.configuration-cache=true
 org.gradle.configureondemand=true
 
 #VERSION_NAME=2023.06.14.101651-421a0ac-SNAPSHOT
+
+org.gradle.parallel=true
```

Comment:
Can this be moved to L9 so it's together with the other settings?

Reply:
Done